demo/mixed-modules/package.json

```diff
@@ -15,11 +15,10 @@
   "author": "koba04",
   "license": "MIT",
   "dependencies": {
-    "google-closure-library": "^20180204.0.0",
+    "google-closure-library": "^20180805.0.0",
```

Comment:
The root package.json uses "google-closure-compiler": ">=20180716.0.1", should we also update that to "^20180805.0.0" to be aligned with the Closure library version?

Reply:
It seems like these will drift over time anyway, as they have historically.